dev/app/builtin/stories/numeric-menu.stories.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/default */
+
+import { storiesOf } from 'dev-novel';
+import instantsearch from '../../../../index';
+import { wrapWithHits } from '../../utils/wrap-with-hits';
+
+const stories = storiesOf('NumericMenu');
+
+export default () => {
+  stories
+    .add(
+      'default',
+      wrapWithHits(container => {
+        window.search.addWidget(
+          instantsearch.widgets.numericMenu({
+            container,
+            attribute: 'price',
+            operator: 'or',
```

Comment:
how does `or` make sense on a menu? A menu can only have one refinement, so can't have an operator (I think?) 🤔

Reply:
Right, that probably was there before. Removing it.